packages/program/src/functions/base-invocation-scope.ts

```diff
@@ -388,4 +392,25 @@ export class BaseInvocationScope<TReturn = any> {
     const transactionRequest = await this.getTransactionRequest();
     return transactionRequest.getTransactionId(chainIdToHash);
   }
+
+  /**
+   * In case the gasLimit and gasPrice are *not* set by the user, this method sets some default values.
+   */
+  async setDefaultTxParams() {
```

Comment:
```suggestion
  private async setDefaultTxParams() {
```
We can make this private as it's an internal detail.